src/stravalib/client.py

```diff
@@ -1004,10 +1007,24 @@ def get_activity_zones(
         :class:`list`
             A list of :class:`stravalib.model.BaseActivityZone` objects.
 
+        Notes
+        -----
+        Activity zones require a Strava premium account.
+
         """
         zones = self.protocol.get("/activities/{id}/zones", id=activity_id)
+        # For troubleshooting
```

Comment:
i would remove this - i just created this look to allow for easier line by line debugging. here the data for the first part of the test (all ints) populates into the model fine. the data for the second set of zones fails because of ints.

Reply:
Supposedly, Pydantic 2 applies stricter validation. It looks like I created that test data from a real Strava API response. It also looks like the API spec is wrong (on the Strava side): the `min` and `max` attributes of `ZoneRange` should be of type `Optional[int | float]` (or `int | float | None`).

We can solve this Strava bug by overriding the type on our model class (just as we did for `BaseActivityZone.type`, also because of a misspecified Strava API).

Reply:
I don't think there's any problem with class attributes named `max`. Since they can only be accessed as attributes of an object, there's no clash with the builtin `max()`.

Reply:
perfect. resolving